code/modules/mob/mob.dm

```diff
@@ -664,7 +664,6 @@
 
 	if(.)
 		if(statpanel("Status") && SSticker.current_state != GAME_STATE_PREGAME)
-			stat("Storyteller", "[master_storyteller]")
```

Comment:
what is this change?

Reply:
I found its cause, https://github.com/discordia-space/CEV-Eris/pull/3293/commits/96843776b453c062758603e951ac6dc7ab5232f3

My pull request is quite outdated, take it as merge conflict. Gotta fix it by myself on next commit, btw

Reply:
or just suggest it
```suggestion
			stat("Storyteller", "[master_storyteller]")
```

Reply:
But I have no powers to accept suggestion there, so you gotta do it yourself or tell me to add commit.